src/sentry/api/endpoints/group_details.py

```diff
@@ -235,6 +236,17 @@ def get(self, request: Request, group) -> Response:
                 )
                 data.update({"sentryAppIssues": sentry_app_issues})
 
+            if "hasAttachments" in expand:
+                if not features.has(
+                    "organizations:event-attachments",
+                    group.project.organization,
+                    actor=request.user,
+                ):
+                    return self.respond(status=404)
+
+                attachments = EventAttachment.objects.filter(group_id=group.id)
+                data.update({"hasAttachments": len(attachments) > 0})
```

Comment:
We could use .count() instead of the len() to save some work inside the db

https://docs.djangoproject.com/en/5.0/topics/db/aggregation/#cheat-sheet

It's possible to inspect the query to be sure, but we only need the db to return the one number. With len() I worry that the db is returning all the rows and columns  into python land.

Reply:
@ryan953 updated to use `count()`